src/cli/index.ts

```diff
@@ -1,89 +1,114 @@
 import * as path from 'path';
 import * as child_process from 'child_process';
+import sade from 'sade';
 import mri from 'mri';
 import chalk from 'chalk';
+import prettyMs from 'pretty-ms';
 import help from './help.md';
 import build from './build';
 import exporter from './export';
 import dev from './dev';
 import upgrade from './upgrade';
+import * as port_utils from './port-utils';
+import { exists } from '../utils';
 import * as pkg from '../../package.json';
 
-const opts = mri(process.argv.slice(2), {
-	alias: {
-		h: 'help'
-	}
-});
+const prog = sade('sapper');
 
-if (opts.help) {
-	const rendered = help
-		.replace('<@version@>', pkg.version)
-		.replace(/^(.+)/gm, (m: string, $1: string) => /[#>]/.test(m) ? $1 : `  ${$1}`)
-		.replace(/^# (.+)/gm, (m: string, $1: string) => chalk.bold.underline($1))
-		.replace(/^> (.+)/gm, (m: string, $1: string) => chalk.cyan($1));
+prog.version(pkg.version);
```

Comment:
Definitely personal preference, but I like to recommend assigning all _global_ aspects into the initial assignment. New commands get their own blocks, and that consistent spacing helps separate things with larger CLIs.

```js
const prog = sade('sapper').version(pkg.version);`
```

Reply:
Ah, nice. I didn't realise it was chainable, I suppose that makes sense!

Reply:
Aye. You can chain the whole thing if you want, but that'll get messy